docs/how-to-guides/curating-the-editor-experience.md

```diff
@@ -12,7 +12,11 @@ Users have the ability to lock and unlock blocks via the editor. The locking UI
 
 ![Image of locking interface](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/Locking%20interface.png?raw=true)
 
-Keep in mind that each block you want to lock will need to be individually locked as desired. There is not a way to mass lock blocks currently. 
+Keep in mind that you can apply locking options to blocks nested inside of a containing block by turning on the "Apply to all blocks inside" option. However, you cannot mass lock blocks otherwise.
```

Comment:
I suggest adding in here that using this option in the UI automatically adds the lock attribute to all the nested blocks. 

Reading it as is, sounds like there is a special attribute you could also add via code which would apply it to all descendants. 

Maybe the "However, you cannot mass lock blocks otherwise" is meant to say just that but I'm not sure that conveys it.